.github/workflows/pdk-validate.yml

```diff
@@ -0,0 +1,15 @@
+name: Run pdk static validation
+
+on:
+  - push
+  - pull_request
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Clone repository
+      uses: actions/checkout@v2
+
+    - name: Run pdk validate
+      uses: puppets-epic-show-theatre/action-pdk-validate@v1
```

Comment:
This looks like a nicely lightweight action for testing :-). 
I'm unsure, though, whether we should look at the workflows shipped as part of https://github.com/puppetlabs/pdk-templates in the long run.

Reply:
Definitely those look much more powerful but also significantly more complex. And Puppetlabs adds the warning:

> These workflows are depending on puppet-internal resources and are currently not suited for public consumption. Feel free to take them as inspiration how to run some tests on Github Actions.

Reply:
Indeed, it seems they rather recommend to use the GitLab CI with GitHub via `.gitlab-ci.yml`. 
Checking out what [voxpopuli](https://github.com/voxpupuli) uses, I find for example this [CI workflow](https://github.com/voxpupuli/puppet-systemd/blob/master/.github/workflows/ci.yml) in [puppet-systemd](https://github.com/voxpupuli/puppet-systemd). They appear to sync these across their repositories via [modulesync](https://github.com/voxpupuli/modulesync_config), i.e. the code comes from [here](https://github.com/voxpupuli/modulesync_config/tree/master/moduleroot/.github/workflows). 

This might be a better fit and seems much more readable at first glance ;-). In any case, that would be an option for the future once we have tests.